### DISCUSSION TITLE
FBW no longer being updated!!!

### DISCUSSION THREAD

**Discussion by usrpc** [flybywiresim/aircraft#6984] [General]
What's Up ???

**Comment by tracernz** [comment]
You can follow the latest updates on the development version at https://github.com/flybywiresim/a32nx/commits/master... In-progress work on the experimental version is a little harder to follow... try https://github.com/flybywiresim/a32nx/pulls?q=is%3Aopen+is%3Apr+label%3A%22Available+on+Experimental%22
